.github/workflows/release.yml

```
@@ -0,0 +1,346 @@
+        run: cross build --release --locked --verbose --target ${{ matrix.target }}
+
+      - name: Build with cargo
+        if: matrix.build_name != 'linux-arm64-musl'
+        run: cargo build --release --locked --verbose --target ${{ matrix.target }}
+
+      - name: Upload binary to release (unix)
+        if: matrix.os != 'windows-latest'
+        uses: softprops/action-gh-release@v2
+        with:
+          files: target/${{ matrix.target }}/release/dotagents
+          asset_name: dotagents-${{ matrix.build_name }}
```

actionlint (1.7.12)

[error] input "asset_name" is not defined in action "softprops/action-gh-release@v2". available inputs are "append_body", "body", "body_path", "discussion_category_name", "draft", "fail_on_unmatched_files", "files", "generate_release_notes", "make_latest", "name", "overwrite_files", "prerelease", "preserve_order", "previous_tag", "repository", "tag_name", "target_commitish", "token", "working_directory"

(action)